spec/controllers/bepstore/goals/goals_controller_spec.rb

```diff
@@ -3,16 +3,12 @@
 RSpec.describe BEPStore::Goals::GoalsController, type: :controller do
   routes { BEPStore::Goals::Engine.routes }
 
-  include_context 'authentication'
-  include_context 'group'
-
-  let!(:parent) { FactoryGirl.create(:activity, groups: [group]) }
-
-  let!(:activity) { FactoryGirl.create(:goal, user: user, groups: [group]) }
-  let!(:activity2) { FactoryGirl.create(:goal, groups: [group]) }
-
-  let!(:params) { { title: "Testing title",url: "https://plus.google.com/_/fake-hangouts-id", group_ids: [group.id], parent_ids: [parent.id] } }
+  let!(:params) do
+    {
+      title: 'Testing title',
+      url: 'https://plus.google.com/_/fake-hangouts-id'
```

Comment:
I think this URL should at the very least be a non-hangout URL, however I think it can be removed entirely from the Goal model and the test as well